apps/src/templates/PaneHeader.jsx

```diff
@@ -18,6 +18,7 @@ class PaneHeader extends React.Component {
   static propTypes = {
     hasFocus: PropTypes.bool.isRequired,
     style: PropTypes.object,
+    isOldPurpleColor: PropTypes.bool,
```

Comment:
Could you add a quick comment for why this prop is needed (and possibly link to the appropriate JIRA/commit/PR like Molly suggested)?